wsmaster/che-core-api-factory-github-common/src/main/java/org/eclipse/che/api/factory/server/github/AbstractGithubPersonalAccessTokenFetcher.java

```diff
@@ -41,7 +41,7 @@ public abstract class AbstractGithubPersonalAccessTokenFetcher
   private final OAuthAPI oAuthAPI;
 
   /** GitHub API client. */
-  private final GithubApiClient githubApiClient;
+  private GithubApiClient githubApiClient;
```

Comment:
My suggestion is to remove `GithubApiClient` from the global variables and instantiate on demand in class methods.

Reply:
Done